### PR TITLE
Feat: MoveGamePlayers

### PR DESCRIPTION
**Pull request by Andrewki44**
Implemented the `!movegameplayers <game_id>` function.
Will automatically move players in an in_progress_game to their assigned voice channels, to speed up team & vc organisation.

Option is disabled by default in .env.example, to ensure member-movements have to be consciously enabled.

### CODE REVIEW

**Comment by Andrewki44** [comment]
To Do:

- Opt in & Opt out per player (with configurable default setting)
- Queue configuration to auto-move players

**Comment by Andrewki44** [comment]
move_enabled added to player & queue (default to 0 / false)
setqueuemoveenabled added
setmoveenabled added

To Do: Implement opt-in vs opt-out.
- Currently is only opt-in

File: discord_bots/commands.py [hunk 1]
```diff
@@ -2540,6 +2502,99 @@ async def mockqueue(ctx: Context, queue_name: str, count: int):
...
+            colour=Colour.red()
+        )
+        return
+    else:
+        await _movegameplayers(game_id, ctx)
+        await send_message(
+        message.channel,
+        embed_description=f"Players moved to voice channels for game {game_id}",
+        colour=Colour.green()
+    )
```
Comment (dwayneyuen) [inline discord_bots/commands.py:2596]: can you see if you have the commit hook setup? look up `darker` in the readme

Comment (Andrewki44) [reply discord_bots/commands.py:2597]: Will get this setup for the future 👍

**Review by dwayneyuen — APPROVED** [review]

**Pull request merged**